cluster/saltbase/README.md

```diff
@@ -0,0 +1,12 @@
+# SaltStack configuration
+
+This is the root of the SaltStack configuration for Kubernetes. A high
+level overview for the Kubernetes SaltStack configuration can be found [in the docs tree.](../../docs/salt.md)
+
+This SaltStack configuration currently applies to default
+configurations for Debian-on-GCE, Fedora-on-Vagrant, Ubuntu-on-AWS and
+Ubuntu-on-Azure. (That doesn't mean it can't be made to apply to an
+arbitrary configuration, but those are only in-tree supported OS/IaaS
+combinations supported today.) As you peruse the configuration, these
```

Comment:
I actually originally had each directory described, then I realized they belonged in readmes in the directory below. Not sure how to make it the most visible.